tests/kvbm_integration/test_consolidator_router_e2e.py

```
@@ -759,15 +867,60 @@ def test_remove_deduplication_across_sources(
+                        logger.error(f"Request {request_idx} failed: {e}")
+                        return (request_idx, False)
+
+                completed_count = 0
+                failed_count = 0
+                with ThreadPoolExecutor(max_workers=concurrency) as executor:
+                    futures = {
+                        executor.submit(send_request, i): i
+                        for i in range(num_requests)
+                    }
+                    for future in as_completed(futures):
+                        request_idx, success = future.result()
```

Ruff (0.14.10)

Unpacked variable `request_idx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)